src/main/resources/jenkins/plugins/ui_samples/Tooltips/index.jelly

```diff
@@ -0,0 +1,46 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:x="jelly:xml">
+  <s:sample>
+    <div class="app-component-sample">
+      <button tooltip="I am a tooltip" class="jenkins-button jenkins-button--primary">
+        Hover over me
+      </button>
+    </div>
+    <pre>
+      <code class="language-xml">
+        <xmp>
+          <button tooltip="I am a tooltip" class="jenkins-button jenkins-button--primary">
+            Hover over me
+          </button>
+        </xmp>
+      </code>
+    </pre>
+
+    <h2>Showing HTML in tooltips</h2>
```

Comment:
is this using a feature from a different branch as it doesn't work for me?

Reply:
Perhaps related to https://github.com/jenkinsci/jenkins/pull/5976? 

I checked the current code of core, there is nothing such `html-tooltip` available as behavior in Jenkins core.

Reply:
probably related to some upcoming work that @janfaracik has in a branch somewhere.

It's a better solution than the above as it explicitly it opted-into.

Although possibly better for a data attribute given my IDE complains that buttons aren't allowed to have this attribute :)

Reply:
totally 👍 for the `data-` hence my comment on the PR ^^